cmd/gosec/main.go

```diff
@@ -147,14 +150,14 @@ func usage() {
 	fmt.Fprint(os.Stderr, "\n\nRULES:\n\n")
 
 	// sorted rule list for ease of reading
-	rl := rules.Generate()
-	keys := make([]string, 0, len(rl))
-	for key := range rl {
+	rl := rules.Generate(false)
```

Comment:
I think here you need to use the `flagTrackSuppressions` instead of hardcoding `false`.